string.js

```diff
@@ -0,0 +1,103 @@
+//  1. `console.log` the first character of a string.
```

Comment:
I saw instances where slice method could be used in better way. I think you understand general concept of string properties. Please keep practicing different methods as string manipulation is crucial for Javascript mastery. Also keep in mind while you are coding that hard coding is very bad practice. Always think about writing a code that can perform same task regardless of your input. Good code can accommodate differing inputs and still results in consistent outputs.